llvm_objects.py

```diff
@@ -162,7 +205,7 @@ def _get_instructions(self, l_block, w_module):
             prev = current_instr
             w_instrs.append(current_instr)
             l_instr = llwrap.LLVMGetNextInstruction(l_instr)
-        return w_instrs
+        return w_instrs[:]
```

Comment:
Aha, I see. I agree that you have to do the copying, but I think it should be done by the person calling `_get_instructions` (since there's no particular reason why it should return an immutable list; it just depends on what the caller wants to do).